### PR TITLE
Renamed project to org.eclipse.xsemantics

### PR DESCRIPTION
**Pull request by mmews-n4**
Github shows three commits here, but actually, only the last one by @qtran-n4 introduces changes.

@LorenzoBettini Please have a thorough at the changes and run your test suites. From our perspective, the state on this branch is ready to be zipped for the initial contribution of the Eclipse/Xsemantics project. Thanks for taking care of the documentation!

### CODE REVIEW

**Comment by LorenzoBettini** [comment]
The Maven groupId is still wrong: net.sf.xsemantics 
It should be updated accordingly. I see that Xtext does something like that:

<groupId>org.eclipse.xtext</groupId>

So I think we could do something similar:

<groupId>org.eclipse.xsemantics</groupId>

File: devtools/org.eclipse.xsemantics.targetplatform/pom.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Contributors:
+  Lorenzo Bettini - Initial contribution and API
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<relativePath>../../releng/org.eclipse.xsemantics.parent/pom.xml</relativePath>
+		<groupId>net.sf.xsemantics</groupId>
```
Comment (LorenzoBettini) [inline devtools/org.eclipse.xsemantics.targetplatform/pom.xml]: The groupId is wrong, see my comment in the Conversation tab

**Comment by qtran-n4** [comment]
@LorenzoBettini I have renamed the groupId to org.eclipse.xsemantics both in xml and documentation. Please check if everything is fine. Thanks!

**Comment by LorenzoBettini** [comment]
Hi, the build runs fine; I can't do any further checks at the moment, but everything looks fine, so I merge the PR and I think you can go on with the zip file :)

**Pull request merged**